charts/forseti-security/README.md

```diff
@@ -35,11 +35,14 @@ helm install --set production=true \
              --name forseti  \
              --set-string serverKeyContents="$(cat forseti-server.json | base64 - -w 0)" \
              --set-string orchestratorKeyContents="$(cat forseti-client.json | base64 - -w 0)" \
+             --set-string serverConfigContents="$(gsutil cat gs://<BUCKET_NAME>/<CONFIGS_FOLDER>/forseti_conf_server.yaml | base64 -)" \
              --values=forseti-values.yaml \
              forseti-security/forseti-security
 ```
 Note that certain values are required.  See the [configuration](#configuration) for details.
 
+Also note that if in a *Darwin* environment, the `-w 0` flag is not supported for the `base64` command and should be ommitted from the above command.
```

Comment:
What is a Darwin environment? Might be better to clarify Mac OS/etc. instead.

Reply:
Darwin is Apple's open source spin of BSD.  Sure, we can change it to mac os